lib/resty/limit/req.lua

```diff
@@ -91,12 +92,7 @@ function _M.incoming(self, key, commit)
         -- we do not handle changing rate values specifically. the excess value
         -- can get automatically adjusted by the following formula with new rate
         -- values rather quickly anyway.
-        excess = tonumber(rec.excess) - rate * abs(elapsed) / 1000 + 1000
-
-        if excess < 0 then
-            -- ngx.log(ngx.WARN, "excess: ", excess / 1000)
-            excess = 0
-        end
+        excess = max(tonumber(rec.excess) - rate * abs(elapsed) / 1000 + 1000, 0)
```

Comment:
This line is now 81 characters... I wanted to avoid:
```lua
excess = tonumber(rec.excess) - rate * abs(elapsed) / 1000 + 1000
excess = max(excess, 0)
```

Please advice :)